olm/operator-source.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: sthaha-operators
```

Comment:
Should we use `openshift-pipelines-operators` here (or something along those lines) ?
Same for the registry namespace, display name and publisher (and in `subscription.yaml`)